src/loaders/it_load.c

```diff
@@ -1009,17 +1018,17 @@ static int load_it_pattern(struct module_data *m, int i, int new_fx,
 			 *                     but not available in the editor)
 			 */
 			switch (b) {
-			case 0xff:	/* key off */
-				b = XMP_KEY_OFF;
-				break;
-			case 0xfe:	/* cut */
-				b = XMP_KEY_CUT;
-				break;
-			default:
-				if (b > 119) {	/* fade */
-					b = XMP_KEY_FADE;
-				} else {
-					b++;	/* note */
+				case 0xff:	/* key off */
+					b = XMP_KEY_OFF;
+					break;
+				case 0xfe:	/* cut */
+					b = XMP_KEY_CUT;
+					break;
+				default:
+					if (b > 119) {	/* fade */
+						b = XMP_KEY_FADE;
+					} else {
+						b++;	/* note */
```

Comment:
unwelcome indentation / whitespace changes

Reply:
There isn't a style guide anywhere, but yes, in libxmp `switch` and `case` go on the same indentation level.